module/zfs/spa.c

```diff
@@ -6911,4 +7136,7 @@ module_param(zio_taskq_batch_pct, uint, 0444);
 MODULE_PARM_DESC(zio_taskq_batch_pct,
 	"Percentage of CPUs to run an IO worker thread");
 
+module_param(vdev_draid_max_rebuild, int, 0644);
```

Comment:
Since rebuild is not specific to draid, it should not have draid in the name for this parameter.